inference-engine/tests/functional/plugin/cpu/single_layer_tests/roi_pooling.cpp

```diff
@@ -12,28 +18,64 @@ using namespace CPUTestUtils;
 namespace CPULayerTestsDefinitions {
 enum ProposalGenerationMode { RANDOM, ULTIMATE_RIGHT_BORDER };
 
-using ROIPoolingCPUTestParamsSet = std::tuple<LayerTestsDefinitions::roiPoolingParamsTuple,
+using roiPoolingShapes = std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>>;
```

Comment:
use `InputShape` per each input
see: https://github.com/openvinotoolkit/openvino/pull/8339/files#diff-2fc89727028fbf8865e6b8742a7840fc5531f4d24ece7daf5c30a8e79d42488aR298-R304

Reply:
Done, thank you